### PR TITLE
Docs/semantic release

### PR DESCRIPTION
**Pull request by daks**
`semantic-release` implementation.

About existing tests, I adapted them to work with the new setups, without looking more deeply about the code used (states) or the tests themself.
I deactivated Fedora builds because they fail, and weren't tested before.

---

Update (from @myii):

- [x] Upgrade the CI matrix to use the pre-salted images -- the current run takes over 20 minutes (over 45 minutes of total time).
- [x] Update the `README` with the `semantic-release` badge and information, including the link to commit message formatting.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,72 @@
...
+
+    - INSTANCE: v2018-3-py2-debian-8
+    - INSTANCE: v2018-3-py2-ubuntu-1604
+    - INSTANCE: v2018-3-py2-bootstrap-centos-6
+    #- INSTANCE: v2018-3-py2-forced-version-fedora-28
+    - INSTANCE: v2018-3-py2-opensuse-423
+
+    - INSTANCE: v2017-7-py2-debian-8
+    - INSTANCE: v2017-7-py2-ubuntu-1604
+    - INSTANCE: v2017-7-py2-bootstrap-centos-6
```
Comment (myii) [inline .travis.yml]: Can we get the new, pre-salted matrix here?

**Review by myii — CHANGES_REQUESTED** [review]
All working well but just a couple of changes as mentioned, if you don't mind.

File: .travis.yml (hunk 1 above)
Comment (daks) [reply .travis.yml]: What? Once again... sorry.

Comment (myii) [reply .travis.yml]: The new pre-salted matrix from the `template-formula`.  Which was introduced with this PR: https://github.com/saltstack-formulas/template-formula/pull/92.

File: docs/README.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
+.. _readme:
+
+cert-formula
+================
+
+|img_travis| |docs| |img_sr|
```
Comment (myii) [inline docs/README.rst]: No `|docs|` at the moment.

File: docs/README.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+.. |img_travis| image:: https://travis-ci.com/saltstack-formulas/cert-formula.svg?branch=master
+   :alt: Travis CI Build Status
+   :scale: 100%
+   :target: https://travis-ci.com/saltstack-formulas/cert-formula
+.. |docs| image:: https://readthedocs.org/projects/docs/badge/?version=latest
+   :alt: Documentation Status
+   :scale: 100%
+   :target: https://cert-formula.readthedocs.io/en/latest/?badge=latest
```
Comment (myii) [inline docs/README.rst]: These four lines to be removed as well.

File: .travis.yml (hunk 1 above)
Comment (myii) [reply .travis.yml]: @daks I'm testing this right now anyway, so the commit is available (temporarily) in my branch:

* https://github.com/myii/cert-formula/commit/5a6ecbc622457dec5cc29029b11475d6d3f6323d

This is currently running **with** Fedora, so let's see what happens with it:

* https://travis-ci.com/myii/cert-formula/builds/109793381

**Comment by daks** [comment]
@myii is the README OK for you?

**Comment by myii** [comment]
@daks Yes, that's fine.  But the pre-salted images aren't being used, as updated in [`kitchen.yml`](https://github.com/saltstack-formulas/template-formula/blob/809207e7f246a1a4e427cb4174e07ca26b4ef4ce/kitchen.yml) in the `template-formula`.

**Review by myii — APPROVED** [review]

**Comment by myii** [comment]
@daks Thanks for the effort, we got there in the end -- merged.

**Pull request merged**